src/Pimple/PimpleContainerAdapter.php

```diff
@@ -48,6 +48,8 @@ public function addServiceConfig(ServiceConfig $config)
     }
 
     /**
+     * @param InflectorConfig $config
```

Comment:
That sounds reasonable. Internally, our rule is simple:

*If PhpStorm doesn't know the type, add just enough documentation to inform it.*

That said, that's all for internal code, so if you think it's better to go with a full docblock for public code then I'm happy to go with that.

The only thing I don't like about documenting types which have a typehint, is the possibility for it to be wrong. e.g.

```php
/**
 * @param $p Foo
 */
function run(Bar $p) { /* ... */ }
```

Feel free to merge ;)